server/src/main/kotlin/suwayomi/tachidesk/global/GlobalAPI.kt

```diff
@@ -20,6 +20,8 @@ object GlobalAPI {
             patch("", GlobalMetaController.modifyMeta)
         }
         path("settings") {
+            get("", SettingsController.serverSettings)
+            patch("", SettingsController.modifyServerSettings)
```

Comment:
probably not possible since it would change an existing endpoint (on the other hand the frontend could quickly be updated), but would it make sense to rename "settings" to "server" and then add a "settings" sub endpoint?
at least I wouldn't consider "about" and "check-update" to be settings

e.g.:
 - server
   - settings
   - about
   - check-update

Reply:
Not a good idea, unnecessary breaking changes are pointless. There might be other private Tachidesk forks and clients we don't know about.